### ISSUE TITLE
Error Installing from Chrome Store "This extension may have been corrupted"

### ISSUE THREAD

**Issue by chadhutchins** [Kocal/vue-web-extension#503]
Hi all, we've developed an extension based on this project. Everything works well when installing in chrome in developer mode and loading from a folder. 

But since adding our extension to the Chrome Store, after a few seconds of use with the browser extension, the icon will disappear and on the chrome extension management page all it shows is "This extension may have been corrupted." There are no other errors listed.

Has anyone else seen this?

Is there anything in developer mode vs. regular mode that is changed?

Any ideas? Thanks!

**Comment by xerosanyam** [comment]
Trying to load a zip in firefox, getting the following error:
![image](https://user-images.githubusercontent.com/3167278/70561148-f3b25980-1baf-11ea-92d3-dc2f65f80151.png)

**Comment by xerosanyam** [comment]
Was adding in firefox in the wrong manner. 
need to add from about:debugging

**Comment by spekulatius** [comment]
Hello @chadhutchins,

is this issue still happening? Have you been able to fix it? If so, how? 

Peter

**Comment by chadhutchins** [comment]
@spekulatius We were able to figure it out! This one was super frustrating and ended up being something pretty random. We set the browser extension popup icon to look different based on the app's state. Our filename reference was not right. The file was lower-case but we were referencing by an upper case.

We changed from this:

`chrome.browserAction.setIcon({ path: 'icons/Red-icon.png' }); // upper case R` 

To: 

`chrome.browserAction.setIcon({ path: 'icons/red-icon.png' }); // lower case r`

Such a random thing to cause as many headaches as it did. It may not be this specific functionality, but check any filename references, make sure spelling/casing is correct.

I hope this helps!
